### PR TITLE
SUP-5591 Switch to use ssh key data instead of local path to ssh key

### PR DESCRIPTION
**Pull request by nguyentr17**
# Update
- Use ssh key data instead of local path to ssh key.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -21,7 +21,7 @@ module "hdinsight_cluster" {
 
   # SSH Access
   username        = var.username
```
Comment (souza-dan) [inline main.tf]: this is probably where terraform fmt is failing.
https://ops.tamr.build/blue/organizations/jenkins/Terraform%20Module%20Repos%2Fterraform-azure-hdinsight-hbase/detail/PR-24/2/pipeline

I find myself running this command quite a lot at the root of terraform repos:
```
terraform fmt --recursive ./
```

**Review by souza-dan — COMMENTED** [review]
Content looks great!

Since this PR removes a variable, we'll consider this a major version.

Let's update the version and changelog to take the repo from v4.0.0 -> v5.0.0.

Here's an example commit of what updating the version and changelog looks like:
https://github.com/Datatamer/terraform-azure-hdinsight-hbase/commit/e24c0d370c90a83f9666ec6a6c7e4c22c8654ea0#diff-7b60b8e351cbb80c47459ffe2c79f1a26404871f49294780fe47ad0e58c09350

We'll want to update the VM module next to keep them consistent.
https://github.com/Datatamer/terraform-azure-tamr-vm

File: main.tf (hunk 1 above)
Comment (nguyentr17) [reply main.tf]: Thanks, I just ran the reformatting and the check passed now!

**Comment by nguyentr17** [comment]
@souza-dan I updated the version in example and changelog. I will submit another SUP ticket for change in tamr-vm and open a corresponding PR.

**Review by souza-dan — APPROVED** [review]
LGTM

**Pull request merged**